src/main/java/org/checkstyle/autofix/recipe/UpperEll.java

```diff
@@ -30,6 +40,16 @@
  */
 public class UpperEll extends Recipe {
 
+    private List<CheckstyleViolation> violations;
+
+    public UpperEll() {
+        this.violations = new ArrayList<>();
+    }
```

Comment:
Why do we need this default constructor? If we do need it, let's update it to `this(new ArrayList<>());`